README.md

```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
+## about Circles
+Tired of using a poorly laid out spreadsheet to cobble together a plan to follow for the next 3-8 years of your life?
+have no fear! Circles is here!
+
+Circles is a UNSW degree planner, which you can use to explore, check, and play around with your course planning.
+
+### features
+<img src="https://raw.githubusercontent.com/csesoc/Circles/dev/frontend/src/images/infographic.png">
+NOTE: we currently only support the Computer Science (3778) degree - but we will add more!
```

Comment:
This section isn't formatting as expected
![image](https://user-images.githubusercontent.com/91037270/166673651-025362ef-55d3-456c-88a3-ddea5cb0871e.png)